include/nbl/builtin/glsl/virtual_texturing/impl_functions.glsl

```diff
@@ -26,9 +26,10 @@ uvec2 nbl_glsl_unpackWrapModes(in uvec2 texData)
 {
     return (texData >> uvec2(28u, 30u))& uvec2(0x03u);
 }
-uint nbl_glsl_unpackMaxMipInVT(in uvec2 texData)
+int nbl_glsl_unpackMaxMipInVT(in uvec2 texData)
 {
-    return (texData.y >> 24) & 0x0fu;
+    uint mm = (texData.y >> 24) & 0x0fu;
+    return mm == 0x0fu ? -1 : int(mm);
```

Comment:
this would be far less painful (and less performance left on the table) if the MaxMipInVT was semantically +1 of what it used to be

Reply:
actually would be even faster to use a +1 semantic but not do a -1 later on